src/main/java/com/adobe/target/edge/client/ClientConfig.java

```diff
@@ -187,120 +205,291 @@ public static final class ClientConfigBuilder {
 
     private ClientConfigBuilder() {}
 
+    /**
+     * Client Code
+     * @param client
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder client(String client) {
       this.client = client;
       return this;
     }
 
+    /**
+     * Organization ID
+     * @param organizationId
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder organizationId(String organizationId) {
       this.organizationId = organizationId;
       return this;
     }
 
+    /**
+     * Server Domain
+     * @param serverDomain
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder serverDomain(String serverDomain) {
       this.serverDomain = serverDomain;
       return this;
     }
 
+    /**
+     * Default Property Token
+     * @param defaultPropertyToken
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder defaultPropertyToken(String defaultPropertyToken) {
       this.defaultPropertyToken = defaultPropertyToken;
       return this;
     }
 
+    /**
+     * Secure (HTTPS) or not
+     * Default value is <b>true</b>
+     * @param secure
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder secure(boolean secure) {
       this.secure = secure;
       return this;
     }
 
+    /**
+     * Socket Timeout
+     * Default value is <b>10000</b>
+     * @param socketTimeout
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder socketTimeout(int socketTimeout) {
       this.socketTimeout = socketTimeout;
       return this;
     }
 
+    /**
+     * Connect Timeout
+     * Default value is <b>10000</b>
+     * @param connectTimeout
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder connectTimeout(int connectTimeout) {
       this.connectTimeout = connectTimeout;
       return this;
     }
 
+    /**
+     * Max Connections Per Host
+     * Default value is <b>100</b>
+     * @param maxConnectionsPerHost
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder maxConnectionsPerHost(int maxConnectionsPerHost) {
       this.maxConnectionsPerHost = maxConnectionsPerHost;
       return this;
     }
 
+    /**
+     * Max Connections Total
+     * Default value is <b>200</b>
+     * @param maxConnectionsTotal
+     * @return ClientConfigBuilder
+     */
     public ClientConfigBuilder maxConnectionsTotal(int maxConnectionsTotal) {
       this.maxConnectionsTotal = maxConnectionsTotal;
       return this;
     }
 
+    /**
```

Comment:
was the indefinite TTL the behavior before these changes?

Reply:
Yes